daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md

```diff
@@ -136,6 +136,20 @@ curl -X POST http://localhost:3500/v1.0/publish/myKafka/myTopic?metadata.partiti
       }'
 ```
 
+### Message headers
+
+All other metadata key/value pairs (that are not `partitionKey`) are set as headers in the Kafka message. For example, here is an example setting a `correlationId` for the message.
+
+```shell
+curl -X POST http://localhost:3500/v1.0/publish/myKafka/myTopic?metadata.correlationId=myCorrelationID \
```

Comment:
Out of interest, would it be worth showing an example of setting two key/value pairs in the message header to show the delimitation?